android/src/main/java/com/aerobotics/DjiMobile/FlightControllerWrapper.java

```diff
@@ -157,15 +166,6 @@ private void uploadMission(final Promise promise) {
       @Override
       public void onResult(DJIError error) {
         if (error == null) {
-          //Sending state update event after delay due to race condition bug in MSDK resulting in incorrect state sometimes being reported
-          new Timer().schedule(
-                  new TimerTask() {
-                    @Override
-                    public void run() {
-                      sendWaypointMissionStateUpdate(getWaypointMissionOperator().getCurrentState());
-                    }
-                  }, 300
-          );
```

Comment:
@MikeWoots like I said in the description, this state update is already sent in the `on<*>Update` callbacks, so choosing not to send another update here.